README.md

```diff
@@ -69,6 +69,10 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [zipe](https://github.com/pikax/zipe) - Server-side renderer.
 
+### Mock
```

Comment:
I feel a bit too much to introduce a new section for "mock". Maybe it's better to be the more general "Testing" or just "Helpers"

Reply:
I think `Testing is better too.